pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -461,13 +277,9 @@ func (pl *dynamicResources) isSchedulableAfterClaimChange(logger klog.Logger, po
 
 	if originalClaim != nil &&
 		originalClaim.Status.Allocation != nil &&
-		originalClaim.Status.Allocation.Controller == "" &&
 		modifiedClaim.Status.Allocation == nil {
 		// A claim with structured parameters was deallocated. This might have made
 		// resources available for other pods.
-		//
-		// TODO (https://github.com/kubernetes/kubernetes/issues/123697):
```

Comment:
Does this mean this issue should get closed?

Reply:
I kept it open because there might be other work related to it. I'll check later.